compiler/rustc_errors/src/lib.rs

```diff
@@ -1131,7 +1136,8 @@ impl HandlerInner {
         let mut diagnostic = Diagnostic::new(Level::Bug, msg);
         diagnostic.set_span(sp.into());
         diagnostic.note(&format!("delayed at {}", std::panic::Location::caller()));
-        self.delay_as_bug(diagnostic)
+        self.delay_as_bug(diagnostic);
+        ErrorReported::new_i_know_what_im_doing()
```

Comment:
This should probably be obtained from `delay_as_bug`?

Reply:
@eddyb Agreed, but currently that returns an `Option<ErrorReported>`...